tags/reserved_tags.go

```diff
@@ -147,6 +147,19 @@ const (
 	XAkitaDumpFilterFlag Key = "x-akita-dump-filter-flag"
 )
 
+// Tags applied to a copy of the spec
+const (
+	XAkitaOriginalOrganization = "x-akita-original-organization"
+
+	XAkitaOriginalService = "x-akita-original-service"
+
+	XAkitaOriginalServiceID = "x-akita-original-service-id"
+
+	XAkitaOriginalName = "x-akita-original-name"
```

Comment:
`XAkitaOriginalSpecName = "x-akita-original-spec-name"`

Reply:
I changed to x-akita-original spec; original-service and original-spec are names; original-service-id and original-spec-id are IDs.